plugin/backend_linux.go

```diff
@@ -679,6 +685,9 @@ func (pm *Manager) CreateFromContext(ctx context.Context, tarCtx io.ReadCloser,
 	}
 
 	tmpRootFSDir, err := ioutil.TempDir(pm.tmpDir(), ".rootfs")
+	if err != nil {
```

Comment:
@runcom Did you want to clean these up? Also it would be good to use `errors.Wrap` anyway to provide extra context.